lab_1_classify_by_unigrams/main.py

```diff
@@ -80,6 +136,15 @@ def preprocess_profile(profile: dict) -> dict[str, str | dict] | None:
     :return: a dict with a lower-cased loaded profile
     with relative frequencies without unnecessary ngrams
     """
+    if not isinstance(profile, dict) or not ('freq' or 'name' or 'n_words') in profile:
+        return None
+    edited_profile = {'name': profile['name'], 'freq': {}}
+    for token in profile['freq']:
+        if token.lower() in edited_profile['freq']:
+            edited_profile['freq'][token.lower()] += profile['freq'][token]/profile['n_words'][0]
```

Comment:
save `token.lower()`, `profile['freq'][token]`, `profile['n_words'][0]` to variables

Reply:
done!